functionary/ui/views/teams/detail.py

```diff
@@ -1,33 +1,24 @@
 from django.contrib.auth.mixins import LoginRequiredMixin, UserPassesTestMixin
 from django.views.generic.detail import DetailView
-from django.views.generic.list import ListView
 
 from core.auth import Permission
-from core.models import Team, TeamUserRole, Variable
-
-
-class TeamListView(LoginRequiredMixin, ListView):
-    model = Team
-
-    def get_queryset(self):
-        """Sorts based on team name, then env name."""
-        if self.request.user.is_superuser:
-            return super().get_queryset().order_by("name")
-        else:
-            teams = TeamUserRole.objects.filter(user=self.request.user).values(
-                "team_id"
-            )
-            return Team.objects.filter(id__in=[team["team_id"] for team in teams])
+from core.models import Team, Variable
 
 
 class TeamDetailView(LoginRequiredMixin, UserPassesTestMixin, DetailView):
     model = Team
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        team = self.get_object()
+        team: Team = self.get_object()
+        context["team_create_perm"] = (
+            True
+            if (self.request.user.has_perm(Permission.TEAM_UPDATE, team))
+            else False
+        )
+        context["team_id"] = str(team.id)
         context["environments"] = team.environments.all()
-        context["users"] = [user_role.user for user_role in team.user_roles.all()]
+        context["users"] = [user for user in team.user_roles.all()]
```

Comment:
Are these user_role's now and not users?

Reply:
They are still user_roles. Just a leftover from a previous refactor of mine

Reply:
No, sorry, you are right. They are user_roles now since I needed to display the user roles on the frontend for the admin users